powertools-idempotency/src/test/java/software/amazon/lambda/powertools/idempotency/internal/IdempotencyAspectTest.java

```diff
@@ -67,17 +71,22 @@ public void firstCall_shouldPutInStore() {
 
         IdempotencyEnabledFunction function = new IdempotencyEnabledFunction();
 
+        when(context.getRemainingTimeInMillis()).thenReturn(30000);
+
         Product p = new Product(42, "fake product", 12);
         Basket basket = function.handleRequest(p, context);
         assertThat(basket.getProducts()).hasSize(1);
         assertThat(function.handlerCalled()).isTrue();
 
         ArgumentCaptor<JsonNode> nodeCaptor = ArgumentCaptor.forClass(JsonNode.class);
-        verify(store).saveInProgress(nodeCaptor.capture(), any());
+        ArgumentCaptor<OptionalInt> expiryCaptor = ArgumentCaptor.forClass(OptionalInt.class);
+        verify(store).saveInProgress(nodeCaptor.capture(), any(), expiryCaptor.capture());
         assertThat(nodeCaptor.getValue().get("id").asLong()).isEqualTo(p.getId());
         assertThat(nodeCaptor.getValue().get("name").asText()).isEqualTo(p.getName());
         assertThat(nodeCaptor.getValue().get("price").asDouble()).isEqualTo(p.getPrice());
 
+        assertThat(expiryCaptor.getValue().orElse(-1)).isEqualTo(30000);
```

Comment:
What is the purpose of `orElse(-1)` here?

Reply:
Because if there is no value, it is an OptionalInt and cannot be compared to an int.